src/rpm_ostree/cli_status.rs

```diff
@@ -226,18 +147,4 @@ mod tests {
             assert_eq!(deployments.len(), 1);
         }
     }
-
-    #[test]
-    fn mock_booted_basearch() {
```

Comment:
These two are some basic unit tests to ensure we don't regress on parsing FCOS-specific metadata, I'd rather not drop it.